app/addons/documents/doc-editor/actions.js

```diff
@@ -141,9 +141,13 @@ function (app, FauxtonAPI, ActionTypes) {
     // store the xhr in parent scope to allow us to cancel any uploads if the user closes the modal
     xhr = $.ajaxSettings.xhr();
 
+    var query = '?rev=' + params.rev;
```

Comment:
it is a query string, but for attachments it will always be the document revision right? either way still +1